Sources/RswiftCore/RswiftCore.swift

```diff
@@ -13,6 +13,17 @@ import XcodeEdit
 public struct RswiftCore {
 
   static public func run(_ callInformation: CallInformation) throws {
+    let lastRunPath = callInformation.tempDir.appendingPathComponent("rswift-lastrun").path
+    guard callInformation.scriptInputFiles.contains(lastRunPath) else {
```

Comment:
This should handle both `$(TEMP_DIR)/rswift-lastrun` and `$TEMP_DIR/rswift-lastrun`

Reply:
This is already working, Xcode expands all variables for us, so we only see the resolved paths and not how the variable is noted.